vendor/models/invoice.py

```diff
@@ -65,7 +65,7 @@ class Meta:
     def __str__(self):
         if not self.profile.user:
             return _("New Invoice")
-        return _("{username} Invoice ({time})").format(username=self.profile.user.username, time=self.created.strftime('%Y-%m-%d %H:%M'))
+        return str(self.profile.user.username) + " Invoice (" + self.created.strftime('%Y-%m-%d %H:%M') + ")"
```

Comment:
No localization inside of the \_\_str\_\_()

Reply:
added get_invoice_display method